src/kontrol/__main__.py

```diff
@@ -121,17 +121,21 @@ def exec_summary(
     accesses_file: Path,
     contract_names: Path | None,
     output_dir_name: str | None,
+    license: str | None,
```

Comment:
By making the `license` default to `UNLICENSED`, this becomes
```suggestion
    license: str,
```